uaclient/entitlements/base.py

```diff
@@ -56,6 +62,20 @@ def description(self) -> str:
         """A sentence describing this entitlement"""
         pass
 
+    @property
+    def help_info(self) -> str:
+        """Help information for the entitlement"""
+        if self._help_info is None:
+            if os.path.exists(DEFAULT_HELP_FILE):
+                with open(DEFAULT_HELP_FILE, "r") as f:
+                    help_dict = yaml.load(f, Loader=yaml.FullLoader)
+            else:
+                help_dict = DEFAULT_HELP
+
+            self._help_info = help_dict.get(self.name, {}).get("help", "")
+
+        return self._help_info
+
```

Comment:
Probably no need to cache as mentioned above.
Let's just read the DEFAULT_HELP_FILE and parse it if it exists. We can follow up on this, with our next PR that actually adds help content, but I think it's probably unnecessary to seed DEFAULT_HELP and rely on a DEFAULT_HELP_FILE. We should go with one or the other, not two ways of doing the same thing. Again, I think we can resolve this in the next PR when we actually update the help content for some of these services.